src/customActions.js

```diff
@@ -112,21 +112,21 @@ CustomActions.prototype.add = function (customAction) {
 };
 
 
-CustomActions.prototype.addScriptLink = function (name, url, scope = "Web") {
+CustomActions.prototype.addScriptLink = function (name, url, scope = "Web", sequence = 100) {
 	var customAction = {
 		Name: name,
 		Title: name,
 		Description: name,
 		Group: name,
-		Sequence: 100,
+		Sequence: sequence,
 		Location: "ScriptLink",
 		Scope: scope,
 		ScriptSrc: url
 	};
 	return this.add(customAction);
 };
 
-CustomActions.prototype.addCSSLink = function (name, url, scope = "Web") {
+CustomActions.prototype.addCSSLink = function (name, url, scope = "Web", sequence = 100) {
```

Comment:
same thing as addScriptLink, use opts object and merge defaults with Object.assign